### PR TITLE
[4.0] Media Manager typo

### PR DESCRIPTION
**Pull request by brianteeman**
COM_MEDIA_ACTIN_RENAME
==>
COM_MEDIA_ACTION_RENAME

Might as well get it right from the beginning or forever have a broken constant

### CODE REVIEW

File: administrator/components/com_media/tmpl/media/default_texts.php [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 use Joomla\CMS\Language\Text;
 
-Text::script('COM_MEDIA_ACTIN_RENAME', true);
+Text::script('COM_MEDIA_ACTION_RENAME', true);
```
Comment (Quy) [inline administrator/components/com_media/tmpl/media/default_texts.php:13]: Sort in alpha order.

**Comment by wilsonge** [comment]
I'm closing this only because I've done this in my media manager a11y PR (yeah I know I suck at single responsibility and I really don't want to rebase it). So formally made that a PR so it doesn't sit in limbo forever

**Comment by brianteeman** [comment]
Grrh

**Comment by infograf768** [comment]
FYI
https://github.com/joomla/joomla-cms/pull/23756#issuecomment-460207089
I have already prepared that RTL patch (also including strings, etc.)
Can modify, but waiting for https://github.com/joomla/joomla-cms/pull/23756 to be completed (tests) and merged.

**Pull request closed without merging**